README.md

```diff
@@ -56,6 +56,7 @@ As the API has just launched, not many implementations exist yet. If you are int
 
 - [SwiftPrometheus](https://github.com/MrLotU/SwiftPrometheus), support for [Prometheus](https://prometheus.io)
 - [StatsD Client](https://github.com/apple/swift-statsd-client), support for StatsD
+- [OpenTelemetry Swift](https://github.com/open-telemetry/opentelemetry-swift), support for [OpenTelemetry](https://opentelemetry.io/) (incl Datadog, Jaeger, Prometheus and Zipkin)
```

Comment:
It does not really make much sense to mention zipkin and jaeger in relation to metrics which this project offers.

How about 
```suggestion
- [OpenTelemetry Swift](https://github.com/open-telemetry/opentelemetry-swift), support for [OpenTelemetry](https://opentelemetry.io/) which also implements other metrics and tracing backends 
```

?

Reply:
You raise a fantastic point that I overlooked when adding this, not all backends make sense in the context of swift-metrics. My apologies! Your wording sounds great, thank you 👍